agent/src/agent/pipeline/config/jython_scripts/influx2_transform_records.py

```diff
@@ -0,0 +1,41 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import sys
+    import os
+    import collections
+
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+
+    from datetime import datetime
+finally:
+    sdc.importUnlock()
+
+DATEFORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+
+def to_timestamp(date):
+    epoch = datetime(1970, 1, 1)
+    return (date - epoch).total_seconds()
+
+
+records = {}
```

Comment:
why do you need a dict here and all those manipulations?
I think just doing `record.value[record.value['_field']] = record.value['_value']` would be enough